parsons/zoom/zoom.py

```diff
@@ -43,17 +80,27 @@ def refresh_header_token(self):
             "content-type": "application/json",
         }
 
-    def _get_request(self, endpoint, data_key, params=None, **kwargs):
-        # To Do: Consider increasing default page size.
+    def _get_request(self, endpoint, data_key, params={}, **kwargs):
+        """
+        TODO: Consider increasing default page size.
+
+        `Args`:
+            endpoint: str
+                API endpoint to send GET request
+            data_key: str
+                Unique value to use to parse through nested data
+                (akin to a primary key in response JSON)
+            params: dict
+                Additional request parameters, defaults to empty dictionary
+
+        `Returns`:
+            Parsons Table of API responses
+        """
 
-        self.refresh_header_token()
         r = self.client.get_request(endpoint, params=params, **kwargs)
         self.client.data_key = data_key
         data = self.client.data_parse(r)
 
-        if not params:
```

Comment:
I think this change to `params` could cause some problems. Currently the public facing functions that call `_get_request` have it default to `None`, not `{}`. That would still result in `params=None` getting passed in, and it won't be reset to an empty dict.

Could that cause issues?